snuba/migrations/runner.py

```diff
@@ -148,6 +149,9 @@ def show_all(
             migration_groups = get_active_migration_groups()
 
         migration_status = self._get_migration_status(migration_groups)
+        clickhouse_group_migrations: MutableMapping[MigrationGroup, List[str]] = {}
+        for group, migration_id in migration_status.keys():
+            clickhouse_group_migrations.setdefault(group, []).append(migration_id)
```

Comment:
nit: Would `defaultdict()` be a better choice here?
```
clickhouse_group_migrations = defaultdict(set)
for group, migration_id in migration_status.keys():
    clickhouse_group_migrations[group].add(migration_id)
```

Reply:
This is probably better yeah